### DISCUSSION TITLE
Some questions about the practical sessions.

### DISCUSSION THREAD

**Discussion by tarook0** [MuhammadSulaiman001/opengl-lab#12] [Q&A] [answered]
Hello Master Muhammad, is there no assignment for the next lecture other than installing the environment and running JoeyDeVries projects? 
Can you give us assignments for the upcoming lectures here?
 I am a student who is repeating this subject and I cannot always attend due to conflicting times with other subjects.

**Comment by MuhammadSulaiman001** [comment, marked as answer]
Hi @tarook0, 

> Can you give us assignments for the upcoming lectures here?

Unfortunately, you have to deliver homeworks in-class (there will be an interview).

> I am a student who is repeating this subject and I cannot always attend due to conflicting times with other subjects.

In your case, you have to attend the lab just for delivering the homeworks.

Notes for regular 3ed-year students announced [here](https://github.com/MuhammadSulaiman001/opengl-lab/discussions/13).
